app/src/shared/components/CommunityStats/Value.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React, { type Node } from 'react'
+import cx from 'classnames'
+import Skeleton from 'react-loading-skeleton'
+
+import Header from './Header'
+
+import styles from './communityStats.pcss'
+
+export type Props = {
+    id: string,
+    label: string | Node,
+    value?: string,
+    unit?: string,
+    loading?: boolean,
+    className?: string
+}
+
+export const Value = ({
+    id,
+    label,
+    value,
+    unit,
+    loading,
+    className,
+}: Props) => (
+    <div key={id} className={cx(styles.stat, className)}>
```

Comment:
Was it extracted from `Values`? I'm asking because of `key={id}`. Is it necessary if in `Values` we already do `key={stat.id}`?